css/properties/animation-timeline.json

```
@@ -141,7 +141,14 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": false
+                "version_added": "114",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "layout.css.scroll-linked-animations.enabled",
```

I thought this was renamed? Wasn't it?

```suggestion
                    "name": "layout.css.scroll-driven-animations.enabled",
```